src/AzureExtension/DevBox/DevBoxInstance.cs

```diff
@@ -684,9 +689,11 @@ public IAsyncOperation<ComputeSystemOperationResult> DoPinActionAsync(string loc
             var psi = new ProcessStartInfo();
             psi.UseShellExecute = true;
             psi.FileName = string.Format(CultureInfo.InvariantCulture, ProtocolPinString, location, pinAction, WorkspaceId, DisplayName, Environment, Username);
+
             Process? p = Process.Start(psi);
             if (p != null)
             {
+                SetForegroundWindow(p.MainWindowHandle);

```

Comment:
If we have one change but not the other, either way, it will simply cause the process to exit with an exitCode indicating the error.  No crashes or exceptions, so we are safe here.